ngi_pipeline/conductor/flowcell.py

```diff
@@ -94,7 +94,7 @@ def process_demultiplexed_flowcells(demux_fcid_dirs, restrict_to_projects=None,
         projects_to_analyze = projects_to_analyze.values()
     for project in projects_to_analyze:
         project_status = CharonSession().project_get(project.project_id)['status']
-        if not project_status != "OPEN":
+        if not project_status == "OPEN":
```

Comment:
YeahI honestly don't know which way BDFL wants me to write this. Should I still be using `not` or just go with `!=`? In any event I should probably not use both. Although that would be a funny way to test for truth.